probability/windowed_median.cpp

```diff
@@ -0,0 +1,213 @@
+/**
+ * @file
+ * @brief An implementation of a median calculation of a sliding window along a
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂

Reply:
The implementation of the algorithm in this contribution is mine, and I couldn't find another source that gives this solution. What I did add was a reference to a solution for the infinite stream problem. My algorithm for the finite window at the back of the stream is inspired by that solution to the infinite stream problem

Reply:
Ahh, I see. That's awesome! Thanks for explaining.